test/engine_tests.jl

```diff
@@ -24,8 +26,18 @@ function solve(file::String)
 end
 
 function get_errors_sets(file::String)::Tuple{Set, Set}
+    # Suppress scoring debug
+    debug = nothing
+    if "JULIA_DEBUG" in keys(ENV)
+        debug = ENV["JULIA_DEBUG"]
+        delete!(ENV, "JULIA_DEBUG")
+    end
     new_errors = get_errors(file)
     old_errors = OldNurseSchedules.get_errors(file)
+    if !isnothing(debug)
+        ENV["JULIA_DEBUG"] = debug
+    end
+
```

Comment:
To supressownie jakoś bardzo dziwnie jest zrobione, logujemy tylko od poziomu `info` do pliku, wiec jak scoring bedzie `debug` to będzie git.